components/automate-ui/src/app/page-components/projects-filter/projects-filter.component.html

```diff
@@ -6,5 +6,6 @@
   [selectionCountActive]="projectsFilter.selectionCountActive$ | async"
   [dropdownCaretVisible]="projectsFilter.dropdownCaretVisible$ | async"
   [filterVisible]="projectsFilter.filterVisible$ | async"
-  (onSelection)="projectsFilter.saveOptions($event)">
+  (onSelection)="projectsFilter.saveOptions($event)"
+  (onOptionChange)="projectsFilter.updateSelectionCount($event)">
```

Comment:
The Output is used to update the number of selected items at the top of the dropdown on each click.  It also updates the dropdown title accordingly.